src/cascadia/TerminalApp/userDefaults.json

```diff
@@ -6,6 +6,12 @@
 
     "defaultProfile": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
 

```

Comment:
Add another comment up here saying something like
```
// You can put more global application settings lines up here!
// Global settings are ones like this: <link to global settings list>
```

because I didn't realize that I could add more of them.